src/wp-content/plugins/cp-pepper/README.md

```diff
@@ -0,0 +1,61 @@
+=== ClassicPress Pepper Password Plugin ===
+Contributors: ClassicPress Team
+Tags: security, users
+Requires at least: 2.0
+Tested up to: 2.2.0
+Requires PHP: 7.2
```

Comment:
Minimum PHP for ClassicPress is 7.4, should the plugin not declare the same version?

Reply:
I can correct that. I used a boilerplate that I do have and forgot to align with plugin ones.